S/SLEEF/bundled/patches/version.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index d16b526..9d4f6b9 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -24,7 +24,7 @@ enable_testing()
+ 
+ set(SLEEF_VERSION_MAJOR 3)
+ set(SLEEF_VERSION_MINOR 4)
+-set(SLEEF_VERSION_PATCHLEVEL 0)
++set(SLEEF_VERSION_PATCHLEVEL 1)
```

Comment:
Why do we need a patch to fix the version number?

Reply:
Because the SLEEF authors forgot to.

Reply:
🤦

Reply:
:-)